shim/src/sanitize.js

```diff
@@ -1,49 +1,7 @@
+import sanitizeAccessors from "./sanitize/accessors";
```

Comment:
I don't like that now we have `sanitizy.js` and `sanitize/*.js`, maybe this folder should not be called `sanitize` but something else were we can drop more similar files in the future.